db/migrate/20150401061243_create_notifications.rb

```diff
@@ -0,0 +1,18 @@
+class CreateNotifications < ActiveRecord::Migration
+  def change
+    create_table :notifications do |t|
+      t.integer :course_id, null: false, foreign_key: { references: :courses }
+      t.integer :user_id, null: false, foreign_key: { references: :users }
+      t.string :action
+      t.string :activity_content
+      t.string :n_type
+      t.string :n_title
+      t.text :n_content
```

Comment:
This should be delegated to the notification for display.

I think we need a proper way to declare notifications, like how CarrierWave declares models?